Library/Homebrew/cask/lib/hbc/dsl.rb

```diff
@@ -204,9 +204,7 @@ def sha256(arg = nil)
     end
 
     def license(*)
-      # TODO: Uncomment after `license` has been
-      #       removed from all official taps.
-      # odeprecated "Hbc::DSL#license"
+      odeprecated "Hbc::DSL#license"
```

Comment:
@reitermarkus Mind committing that to this PR? Not sure where the relevant part is for Cask.